wit/wasi-filesystem.wit.md

```diff
@@ -226,89 +226,87 @@ record dir-entry {
 }
 ```
 
-## `errno`
+## `error-code`
 ```wit
 /// Error codes returned by functions.
 /// Not all of these error codes are returned by the functions provided by this
 /// API; some are used in higher-level library layers, and others are provided
 /// merely for alignment with POSIX.
-enum errno {
+enum error-code {
     /// Permission denied.
     access,
     /// Resource unavailable, or operation would block.
     again,
```

Comment:
What would you think about changing `again` to `would-block`? In POSIX, `EAGAIN` is permitted to be equal to `EWOULDBLOCK` and the errors are the same on many popular platforms. And `would-block` is more descriptive of what the code is for than `again`.

Reply:
Indeed, `would-block` is much more descriptive, I have updated the PR accordingly.